algoliasearch/build.gradle

```diff
@@ -33,10 +33,11 @@ ext {
     - Log-in on sonatype.org, go to staging profile, select com.algolia, id = last part of URL */
 }
 
-
-nexusStaging {
-    packageGroup = PUBLISH_GROUP_ID
-    stagingProfileId = PUBLISH_STAGING_ID
+if (project.hasProperty("nexus")) {
```

Comment:
Btw, have it failed when it was put in the root `build.gradle` (I have not experience with Android apps)?